src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -914,6 +918,10 @@
           type: string
           short-summary: Public key path or key contents to install on node VMs for SSH access. For example,
                          'ssh-rsa AAAAB...snip...UcyupgH azureuser@linuxvm'.
+        - name: --custom-ca-trust-certificates --ca-certs
```

Comment:
```suggestion
        - name: --ca-certs --custom-ca-trust-certificates
```
Maybe change the order and then give it a try?